tests/data/small_sky/properties

```diff
@@ -2,13 +2,13 @@
 obs_collection=small_sky
 dataproduct_type=object
 hats_nrows=131
-hats_col_j2000_ra=ra
-hats_col_j2000_dec=dec
+hats_col_ra=ra
+hats_col_dec=dec
 hats_max_rows=1000000
 hats_order=0
 moc_sky_fraction=0.08333
-hats_builder=hats-import v0.3.6.dev22+g6346e4d
-hats_creation_date=2024-09-30T13\:32UTC
-hats_estsize=98328
+hats_builder=hats-import v0.3.6.dev26+g40366b4
+hats_creation_date=2024-10-11T18\:18UTC
+hats_estsize=49177
```

Comment:
Efficiency gain or reduction in scope?

Reply:
Efficiency gain (that took me a while to track down!).

Nearly all (99.9%, 49160/49208, according to `du`) of the space for this teeeeny catalog is taken up by the `point_map.fits` file, and we recently (PR #361) switched this file to store `dtype=np.int32` instead of int64.

For catalogs in the wild, the ratio is not nearly so off-balance.